config/resources/newsletter-domain.lisp

```diff
@@ -1,50 +0,0 @@
-(define-resource newsletter-info ()
-  :class (s-prefix "besluitvorming:NieuwsbriefInfo")
-  :properties `((:text                  :string   ,(s-prefix "besluitvorming:inhoud"))
-                (:richtext              :string   ,(s-prefix "ext:htmlInhoud"))
-                (:subtitle              :string   ,(s-prefix "dbpedia:subtitle"))
-                (:mandatee-proposal     :string   ,(s-prefix "ext:voorstelVan"))
```

Comment:
5000 hits in data. predicate not cleaned up (that I could find)

Reply:
Ik heb een story gemaakt om dit op te volgen: https://kanselarij.atlassian.net/browse/KAS-3802